test/testUtils/testUtils.py

```diff
@@ -30,6 +30,10 @@ def assertVectorAlmostEqual(testCase, a, b, places=7):
     for va, vb in zip(a, b):
         testCase.assertAlmostEqual(va, vb, places)
 
+def assertVectorNotAlmostEqual(testCase, a, b, places=7):
```

Comment:
Thought I would need this, ended up not using it, but left it in, as it's useful.